certif/tests/acceptance/routes/authenticated/team/invite_test.js

```diff
@@ -0,0 +1,188 @@
+import { module, test } from 'qunit';
+import { setupApplicationTest } from 'ember-qunit';
+import setupMirage from 'ember-cli-mirage/test-support/setup-mirage';
+import { currentURL, visit } from '@ember/test-helpers';
+import { clickByName, fillByLabel } from '@1024pix/ember-testing-library';
+
+import setupIntl from '../../../../helpers/setup-intl';
+import {
+  authenticateSession,
+  createCertificationPointOfContactWithTermsOfServiceAccepted,
+} from '../../../../helpers/test-init';
+
+module('Acceptance | Routes | Team | Invite', function (hooks) {
+  setupApplicationTest(hooks);
+  setupMirage(hooks);
+  setupIntl(hooks);
+
+  module('when user is an admin', function (hooks) {
+    let userWithAdminRole;
+
+    hooks.beforeEach(async function () {
+      userWithAdminRole = createCertificationPointOfContactWithTermsOfServiceAccepted(
+        undefined,
+        'PIX Certification Center',
+        false,
+        'ADMIN',
+      );
+      await authenticateSession(userWithAdminRole.id);
+    });
+
+    test('sends one invitation', async function (assert) {
+      // given
+      const inputLabel = this.intl.t('pages.team-invite.input-label');
+      const inviteButtonLabel = this.intl.t('pages.team-invite.invite-button');
+      const email = '   flavie.eich ouette@example.net           ';
```

Comment:
Même si le code permet de corriger `flavie.eich ouette@example.net` en `flavie.eichouette@example.net` c'est un détail d'implémentation, cela n'est pas dans les specs de ce qu'il y a à réaliser et donc cela n'a pas à apparaitre dans les tests : 

```suggestion
      const email = '   flavie.eichouette@example.net           ';
```